corehq/apps/reports/views.py

```diff
@@ -790,6 +793,10 @@ def scheduled_report_form(self):
                                         (Domain.get_by_name(self.domain)['default_timezone'],
                                         get_timezone_difference(self.domain)[:3] + ':'
                                         + get_timezone_difference(self.domain)[3:])
+        form.fields['stop_hour'].help_text = "This scheduled report's timezone is %s (%s GMT)" % \
+                                        (Domain.get_by_name(self.domain)['default_timezone'],
+                                        get_timezone_difference(self.domain)[:3] + ':'
+                                        + get_timezone_difference(self.domain)[3:])
```

Comment:
It seems like the result of this function is always split into the hour and minute parts. It would be nice if that was handle in the function itself and it returned a dataclass that could be used to access the parts or the formatted string with the ':'